src/legacy/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -458,6 +458,12 @@ export function getUiSettingDefaults() {
            away or update the query. When enabled, dashboard panels will load together when all of the data is loaded, and
            searches will not terminate.`,
       }),
+      deprecation: {
```

Comment:
Would you mind adding it to https://github.com/elastic/kibana/blob/030929e51c3595f29c49a0849bde4ac92e8a2622/src/core/server/ui_settings/types.ts#L86 ?